Formula/u-boot-tools.rb

```diff
@@ -26,8 +27,9 @@ def install
     # Replace keyword not present in make 3.81
     inreplace "Makefile", "undefine MK_ARCH", "unexport MK_ARCH"
 
-    system "make", "sandbox_defconfig"
-    system "make", "tools", "NO_SDL=1"
+    ENV["SOURCE_DATE_EPOCH"] = ""
```

Comment:
This is already set by brew

Reply:
without this setting I get this error message:
```
...
  GEN     include/autoconf.mk
/bin/sh: line 0: return: can only `return' from a function or sourced script
  UPD     include/config/uboot.release
make: *** [include/generated/timestamp_autogenerated.h] Error 1
...
```
because of this:
[u-boot Makefile](https://github.com/u-boot/u-boot/blob/85ccbf666e549f0b06c29d565b9e4fdd87cf6600/Makefile#L1910)

Reply:
You can fix that by adding
```
depends_on "coreutils" => :build # Makefile needs `gdate`
```
to provide `gdate` rather than overriding environment variables we set.

Reply:
thank you, that really looks better